SUMMARY.md

```diff
@@ -26,26 +45,5 @@
     - [This seems really cool?](faq/misc/cool.md)
     - [Who wrote this FAQ?](faq/misc/faq-credits.md)
     - [How can I help?](faq/misc/help.md)
-* [Stories](stories/index.md)
-  - [Design Challenge: Avoid Centralization and Singletons](stories/design-challenge-avoid-centralization-and-singletons.md)
```

Comment:
oh yeah ... that gotcha ... sigh.
Maybe we do put some in the menu but under subheadings like unpublished ... 

it's messy but necessary ?